example/main.jsonnet

```diff
@@ -1,40 +1,38 @@
-local dex = (import 'github.com/observatorium/deployments/components/dex.libsonnet') + {
-  config+:: {
-    local cfg = self,
-    name: 'dex',
-    namespace: 'dex',
-    config+: {
-      oauth2: {
-        passwordConnector: 'local',
-      },
-      staticClients: [
-        {
-          id: 'test',
-          name: 'test',
-          secret: 'ZXhhbXBsZS1hcHAtc2VjcmV0',
-        },
-      ],
-      enablePasswordDB: true,
-      staticPasswords: [
-        {
-          email: 'admin@example.com',
-          // bcrypt hash of the string "password"
-          hash: '$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W',
-          username: 'admin',
-          userID: '08a8684b-db88-4b73-90a9-3cd1661f5466',
-        },
-      ],
-    },
-    version: 'v2.24.0',
-    image: 'quay.io/dexidp/dex:v2.24.0',
-    commonLabels+:: {
-      'app.kubernetes.io/instance': 'e2e-test',
+local dex = (import 'github.com/observatorium/deployments/components/dex.libsonnet')({
+  local cfg = self,
+  name: 'dex',
+  namespace: 'dex',
+  config+: {
+    oauth2: {
+      passwordConnector: 'local',
     },
+    staticClients: [
+      {
+        id: 'test',
+        name: 'test',
+        secret: 'ZXhhbXBsZS1hcHAtc2VjcmV0',
+        issuerCAPath: '/var/run/tls/test/service-ca.crt',
+      },
+    ],
+    enablePasswordDB: true,
+    staticPasswords: [
+      {
+        email: 'admin@example.com',
+        // bcrypt hash of the string "password"
+        hash: '$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W',
+        username: 'admin',
+        userID: '08a8684b-db88-4b73-90a9-3cd1661f5466',
+      },
+    ],
   },
-};
-
-local obs = (import 'github.com/observatorium/deployments/environments/base/observatorium.jsonnet');
+  version: 'v2.24.0',
+  image: 'quay.io/dexidp/dex:v2.24.0',
+  commonLabels+:: {
+    'app.kubernetes.io/instance': 'e2e-test',
+  },
+});
 
+local obs = (import 'github.com/observatorium/deployments/components/observatorium.libsonnet');
```

Comment:
I'd really move all imports to the top. 
Hasn't been that way before, but that's generally what we try to do - making it simliar to Go and now have imports all over the place.

Reply:
Obviously can be done in another PR or never if not deemed important enough.